package.json

```diff
@@ -25,7 +26,7 @@
     "react-tap-event-plugin": "^2.0.1",
     "react-url-query": "^1.1.4",
     "surge": "^0.19.0",
-    "swipe-js-iso": "^2.0.3"
+    "swipe-js-iso": "^2.0.3",
```

Comment:
The Travis build fails because of the comma I guess. Remove comma for the last item in the object.